Documentation/configuration/metrics.rst

```diff
@@ -50,6 +53,9 @@ Policy Imports
 --------------
 
 * ``policy_count``: Number of policies currently loaded
+* ``policy_regeneration_count``: Total number of policies regenerated successfully
```

Comment:
Metrics for unit-less counts should be named `*_total`, cf. https://prometheus.io/docs/practices/naming/

Reply:
done